### PR TITLE
Mac build instructions

### PR DESCRIPTION
**Pull request by slipher**
@DolceTriade see if these work for you.

### CODE REVIEW

**Comment by DolceTriade** [comment]
Will test tomorrow.

File: README.md [hunk 1]
```diff
@@ -51,3 +53,31 @@ docker rm updater2-win
...
+
+### Build Qt
+```
+curl -LO https://download.qt.io/archive/qt/5.14/5.14.2/single/qt-everywhere-src-5.14.2.tar.xz
+tar -xzf qt-everywhere-src-5.14.2.tar.xz
+cd qt-everywhere-src-5.14.2
+MODULES=qtbase,qtquickcontrols,qtquickcontrols2,qtsvg,qtgraphicaleffects
+# Default install location (--prefix): /usr/local/Qt-5.14.2
+./configure -opensource -confirm-license -release -optimize-size -no-shared -no-framework -static --c++std=14 -nomake tests -nomake tools -nomake examples
+eval make -j(whatever) module-{$MODULES}
```
Comment (DolceTriade) [inline README.md]: $(sysctl -n hw.logicalcpu)

**Review by DolceTriade — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (illwieckz) [reply README.md]: Is `hw.logicalcpu` better than `hw.ncpu`, I guess the second one only list cores and while first one also list threads?

Comment (DolceTriade) [reply README.md]: ya, if you have hyperthreading.

**Pull request merged**